src/local/wrapper.sh

```diff
@@ -3,7 +3,7 @@
 #SBATCH --partition $PARTITION_TYPE # one of gpu, gpu_devel
 #SBATCH --nodes 1 # amount of nodes allocated 
 #SBATCH --time 6:00:00 # hh:mm:ss, walltime 
-#SBATCH --mem $MEM 
+##SBATCH --mem $MEM 
```

Comment:
This breaks the wrapper, remove the extra comment?

Reply:
if I launch wrapper.sh with sbatch and #SBATCH --mem $MEM, it complains about invalid flag or value.